queue_job/models/base.py

```diff
@@ -205,3 +205,36 @@ def _job_store_values(self, job):
         :return: dictionary for setting job values.
         """
         return {}
+
+    @api.model
+    def _job_prepare_context_before_enqueue_keys(self):
+        """Keys to keep in context of stored jobs
+        <<<<<<< HEAD
```

Comment:
left over from bad rebase ;)

Reply:
thanks for pointing out. My bad. Fixed now.